WordPressKit/WordPressOrgXMLRPCApi.swift

```diff
@@ -254,6 +282,33 @@ open class WordPressOrgXMLRPCApi: NSObject {
         return progress
     }
 
+    /// Call an XMLRPC method.
+    ///
+    /// ## Error handling
+    ///
+    /// Unlike the closed-based APIs, this method returns a concrete error type. You should consider handle the errors
+    /// as they are, instead of casing them to `NSError` instance. But in case you do need to cast them to `NSError`,
+    /// considering using the `asNSError` function if you need backward compatiblity with existing code,
+    ///
+    /// - Parameters:
+    ///   - streaming: set to true if there are large data (i.e. uploading files) in given `parameters`. `false` by default.
```

Comment:
```suggestion
    ///   - streaming: set to `true` if there are large data (i.e. uploading files) in given `parameters`. `false` by default.
```

"false" at the end is code fenced, so "true" should be, too.